src/core/getFrameAccountAddress.ts

```diff
@@ -20,16 +15,7 @@ async function getFrameAccountAddress(
   { NEYNAR_API_KEY = 'NEYNAR_API_DOCS' },
 ) {
   let farcasterID = 0;
-  let validatedMessage: Message | undefined = undefined;
-  // Get the message from the request body
-  const frameMessage: Message = Message.decode(
-    Buffer.from(body?.trustedData?.messageBytes ?? '', 'hex'),
-  );
-  // Validate the message
-  const result = await client.validateMessage(frameMessage);
-  if (result.isOk() && result.value.valid && result.value.message) {
-    validatedMessage = result.value.message;
-  }
+  const validatedMessage = await parseFrameMessage(body);
```

Comment:
maybe we want to make this a param? That way if you're parsing the message to get other data anyway you can just pass the result in to this function rather than loading same data twice. Could do this with an overload